### PR TITLE
Fix #894 - Memory leak issue

### LINKED ISSUE #894: [Bug][Lake] Memory issue while creating bronze tables with big datasets
We get a memory issue while creating bronze tables
image

image

### PR DESCRIPTION
**Pull request by kdetry**
Fixes #894 

Changes proposed in this PR:

- the split part logic on queries moved to raw tables

### CODE REVIEW

**Review by idiom-bytes — APPROVED** [review]
This looks ok, does the new memory limits work on your setup @KatunaNorbert

File: pdr_backend/lake/table_bronze_pdr_predictions.py [hunk 1]
```diff
@@ -46,8 +46,7 @@ def get_bronze_pdr_predictions_data_with_SQL(
         f"""
         SELECT
             {pdr_predictions_table_name}.ID as ID,
-            SPLIT_PART({pdr_predictions_table_name}.ID, '-', 1)
-                || '-' || SPLIT_PART({pdr_predictions_table_name}.ID, '-', 2) AS slot_id,
+            {pdr_predictions_table_name}.pdr_slot_slot_id as slot_id,
```
Comment (KatunaNorbert) [inline pdr_backend/lake/table_bronze_pdr_predictions.py:49]: This line is trowing me an error. We try to access "pdr_slot_slot_id" from `pdr_predictions` but that field it's not set

File: pdr_backend/subgraph/subgraph_payout.py [hunk 2]
```diff
@@ -163,6 +163,7 @@ def fetch_payouts(
                 ),
                 "roundSumStakes": float(payout["prediction"]["slot"]["roundSumStakes"]),
                 "stake": float(payout["prediction"]["stake"]),
+                "pdr_slot_slot_id": f"{payout['id'].split('-')[1]}-{payout['id'].split('-')[2]}",
```
Comment (KatunaNorbert) [inline pdr_backend/subgraph/subgraph_payout.py:166]: The sot id format is `contract-slot` so the actual value is `"pdr_slot_slot_id": f"{payout['id'].split('-')[0]}-{payout['id'].split('-')[1]}"`

**Comment by KatunaNorbert** [comment]
Did some fixes and tested it. The issue still happens and it's not solved.
The reason it worked was because the slot_id values stored were wrong and when computing the bronze tables the computation load was load because values were not found.
After fixing the slot id value the issues still occurs, feel free to test it

**Pull request closed without merging**